tests/requirements.txt

```diff
@@ -5,3 +5,5 @@ virtualenv>=13.0.0
 pytest-virtualenv>=1.2.7
 pytest>=3.0.2
 wheel
+coverage
```

Comment:
shouldn't this be version qualified with a lower bound; I mean the interface of pytest coverage e.g. changed over time; so I don't think this will run with any version

Reply:
Good idea.